stable/mysql/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.configurationFiles }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+data:
+{{- range $key, $val := .Values.configurationFiles }}
+  {{ $key }}: |-
```

Comment:
The file gets rendered correctly even with those characters. I verified and deployed the chart on my testing k8s cluster using the same configuration as in the README file and it works fine.